packages/flame/src/Popover/Popover.stories.tsx

```diff
@@ -1,18 +1,26 @@
 import React, { Component } from 'react';
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-import { withReadme } from 'storybook-readme';
 import { css } from '@styled-system/css';
+// eslint-disable-next-line
```

Comment:
What is the error that was turned off here?

Reply:
storybook and monorepo shenanigans again. I would need to add storybook stuff to the devDep, but that's technically not needed because the dep is included at the root of the monorepo and stories aren't even bundled in the final build

Reply:
That's fine - non-blocking: I would add what was disabled here, I presume `import/no-extraneous-dependencies`?